packages/pigeon/mock_handler_tester/test/message.dart

```diff
@@ -1,173 +1,148 @@
-// Autogenerated from Pigeon (v0.1.2), do not edit directly.
+// Autogenerated from Pigeon (v0.1.15), do not edit directly.
 // See also: https://pub.dev/packages/pigeon
 // ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import
+// @dart = 2.8
 import 'dart:async';
+import 'dart:typed_data' show Uint8List, Int32List, Int64List, Float64List;
 import 'package:flutter/services.dart';
 
 class SearchReply {
   String result;
   String error;
+
   // ignore: unused_element
-  Map<dynamic, dynamic> _toMap() {
-    final Map<dynamic, dynamic> pigeonMap = <dynamic, dynamic>{};
+  Object encode() {
+    final Map<Object, Object> pigeonMap = <Object, Object>{};
     pigeonMap['result'] = result;
     pigeonMap['error'] = error;
     return pigeonMap;
   }
 
   // ignore: unused_element
-  static SearchReply _fromMap(Map<dynamic, dynamic> pigeonMap) {
-    if (pigeonMap == null) {
-      return null;
-    }
-    final SearchReply result = SearchReply();
-    result.result = pigeonMap['result'];
-    result.error = pigeonMap['error'];
-    return result;
+  static SearchReply decode(Object message) {
+    final Map<Object, Object> pigeonMap = message as Map<Object, Object>;
+    return SearchReply()
```

Comment:
`_fromMap` used to accept null input, now decode doesn't.  I see there appears to be one place where decode is called without a null check to the argument.  Are you sure this is safe to remove?

Reply:
I must have misread the code; it looked like it used to crash when the message was null, but I see in the output for video_player that there's a null check.

I'm happy to make it accept null again.

(Is there ever a valid reason to send null?)

Reply:
Oh I remember why I made it not accept null -- it's not so much that the input is non-nullable, so much as that the _output_ is non-nullable. With a null input I don't know what to return. I could return null but then I need to null check around every call site anyway, and it seemed cleaner to have the null check before the call than after the call.

Reply:
Yea, that makes sense.  I just checked the output with nullsafety and it looks like this:
```
  static SearchReply _fromMap(Map<dynamic, dynamic> pigeonMap) {
    final SearchReply result = SearchReply();
    result.result = pigeonMap['result'];
    result.error = pigeonMap['error'];
    return result;
  }
```

Looks like I already got rid of the null check, this generated code just hasn't been updated in a while.